generate.py

```diff
@@ -30,11 +30,11 @@ def main():
     """
     Generate text/sample model
     """
-    model, neox_args = setup_for_inference_or_eval(use_cache=True)
-    if neox_args.recompute:
-        model.module.inference_mode(
-            use_cache=False
-        )  # don't use kv cache if recomputing
+    model, neox_args = setup_for_inference_or_eval(use_cache=False)
+    # if neox_args.recompute:
+    #     model.module.inference_mode(
+    #         use_cache=False
+    #     )  # don't use kv cache if recomputing
```

Comment:
Can we allow caching? This will espcially be useful when doing longer generations and in context learning?